static/js/user_status_ui.js

```diff
@@ -162,31 +144,51 @@ export function initialize() {
         update_button();
     });
 
-    $("body").on("click", "#set_user_status_modal .set_user_status", () => {
+    submit_button().on("click", () => {
         submit_new_status();
     });
 
-    $("body").on("shown.bs.modal", "#set_user_status_modal", () => {
-        const field = input_field();
-        field.trigger("focus");
-    });
```

Comment:
It would be better if we focus on input field after opening the modal for better keyboard experience.

Reply:
I will be making a new commit that adds support for modal event handlers soon. I will do this in that commit.

Reply:
Let's do that before merging this one -- having the keyboard focus work correctly for widgets like this is important.

Reply:
@timabbott  #20321

Reply:
Great, merged, you can rebase this to use that now.